packages/ripple-address-codec/package.json

```diff
@@ -28,5 +28,10 @@
   "prettier": "@xrplf/prettier-config",
   "engines": {
     "node": ">= 10"
+  },
+  "devDependencies": {
```

Comment:
Why were these dependencies added?

Reply:
@ckniffen `npm run test` didn't work without them. I can back them out if this doesn't belong here.

Reply:
I think they should be added via lerna when doing `npm install`. @justinr1234 is this correct?

Reply:
I verified that if I do `npm install` in the project root this is taken care of. Consequently I reverted the additional `devDependencies`.

Reply:
Yes you can follow instructions here: https://futurestud.io/tutorials/lerna-install-dependencies-for-a-specific-package

`lerna add hash.js --scope=ripple-address-codec`

But I believe the effect might be similar.

Reply:
I did the above with no changes to the local setup, so I think this is resolved.